### ISSUE TITLE
New a11y checker rules causing tests to fail

### ISSUE THREAD

**Issue by matthewgallo** [carbon-design-system/ibm-products#2808]
## What will this achieve?

@janhassel shared a [PR](https://github.com/IBMa/equal-access/pull/1329) from the accessibility check that changed some rules causing a handful of security tests to fail, thus causing the checks on the more recent PRs in our repo to fail as well.

## How will success be measured?

When there are no failing a11y tests with the latest accessibility checker. See [here](https://github.com/carbon-design-system/ibm-cloud-cognitive/actions/runs/4600647950/jobs/8127510641?pr=2804) for an example.

@SimonFinney's already pinned to the last passing version which will get builds passing again until this is addressed more specifically. 🎉 🥇 

## Additional information

- Designs
- Existing code
- etc

**Comment by elycheea** [comment]
Closing as not planned since this package is no longer supported in our v2 and our main package is currently on `latest` ruleset.
